.github/workflows/pull-request.yml

```
@@ -243,51 +243,20 @@
-
-      - name: Merge Jest and Vitest Coverage Reports
-        run: |
-          mkdir -p ./coverage
-          npx lcov-result-merger './coverage/*/lcov.info' './coverage/lcov.info'
   
-      - name: Upload Combined Coverage to Codecov
+      - name: Upload Coverage to Codecov
         uses: codecov/codecov-action@v4
         with:
           token: ${{ secrets.CODECOV_TOKEN }}
           gcov_ignore: 'docs/'
```

actionlint (1.7.4)

input "gcov_ignore" is not defined in action "codecov/codecov-action@v4". available inputs are "codecov_yml_path", "commit_parent", "directory", "disable_file_fixes", "disable_safe_directory", "disable_search", "dry_run", "env_vars", "exclude", "fail_ci_if_error", "file", "files", "flags", "git_service", "handle_no_reports_found", "job_code", "name", "network_filter", "network_prefix", "os", "override_branch", "override_build", "override_build_url", "override_commit", "override_pr", "plugin", "plugins", "report_code", "root_dir", "slug", "token", "url", "use_legacy_upload_endpoint", "use_oidc", "verbose", "version", "working-directory"

(action)